lib/encoding.neon

```diff
@@ -0,0 +1,159 @@
+%|
+ |  File: encoding
+ |
+ |  Functions for encoding and decoding ASCII or Binary data in various formats.
+ |%
+IMPORT bitwise
+
+EXPORT base64Encode
+EXPORT base64Decode
+
+TYPE Mode IS ENUM
+    BYTE1
```

Comment:
Also, as a matter of style, don't use all uppercase here (even though you can in this case because it contains a digit). Neon style is to use lower case for enum values.